state-transition/core/state_processor_payload_test.go

```diff
@@ -142,11 +143,19 @@ func TestPayloadTimestampVerification(t *testing.T) {
 				WithVerifyResult(false).
 				WithMeterGas(false)
 
+			var depRoot common.Root
```

Comment:
technically this is what happens in prod:
- we finalize a block and pull its deposits
- we expect those deposits to be included and processed in the next block

I started these UTs before clearly understanding this